### PR TITLE
testing/bcc: split into subpackages

### PR DESCRIPTION
**Pull request by acj**
~Now that libbpf has releases, I'm including it as part of `source` to simplify the build and patches.~ See discussion below.

### CODE REVIEW

**Comment by kaey** [comment]
pkg-config files should be in -dev.
bcc-tools should also depend on py-bcc.
Everything else looks good.

**Comment by acj** [comment]
Fixed. Thanks.

**Comment by acj** [comment]
@maxice8 @Ikke Sorry for the direct ping, but if you have time for a review, I would love to get this merged. It's blocking work on several downstream projects.

File: testing/bcc/APKBUILD [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+pkgrel=1
 pkgdesc="A toolkit for creating efficient kernel tracing and manipulation programs"
 url="https://github.com/iovisor/bcc/"
 arch="aarch64 x86 x86_64"
 license="Apache-2.0"
 # bcc's test suite requires privileged access to run BPF programs
 options="!check"
 depends="python"
+subpackages="$pkgname-dev:_dev $pkgname-doc:_doc $pkgname-tools:_tools $pkgname-lua:_lua py-$pkgname:_py"
+_libbpfver=0.0.4
```
Comment (maxice8) [inline testing/bcc/APKBUILD]: should libbpf be a separate APKBUILD ?

Comment (acj) [reply testing/bcc/APKBUILD]: I had hoped to simplify the build, but I just noticed that there's been a recent effort to make an RPM package for libbpf. We will probably want one as well.

Force-pushed a change to continue using testing/libbpf-dev.

File: testing/bcc/APKBUILD [hunk 2]
```diff
@@ -33,6 +34,28 @@ build() {
 package() {
 	cd "$builddir/build"
 	make install DESTDIR="$pkgdir"
+
+	mv "$pkgdir"/usr/share/bcc/man "$pkgdir"/usr/share/man
```
Comment (kaey) [inline testing/bcc/APKBUILD]: Tools which these man pages refer to are not in PATH, so it's better to leave them in `/usr/share/bcc/man`.
Furthermore in a shared space they would conflict with man pages provided by bpftrace

**Pull request closed without merging**